src/core/lib/huo_zi_yin_shua.py

```
@@ -45,10 +46,47 @@ def _load_audio(cls, target_path: str) -> np.ndarray:
+                with av.open(out_path, 'w', 's16le') as out_container:
+                    out_stream = out_container.add_stream(
+                        'pcm_s16le',
+                        rate=sample_rate,
+                        layout='mono'
+                    )
+                    for frame in in_container.decode(in_stream):
+                        frame.pts = None
+                        for packet in out_stream.encode(frame):
+                            out_container.mux(packet)
+                    converted = True
+        except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)